### DISCUSSION TITLE
Release 1.19

### DISCUSSION THREAD

**Discussion by webspiderteam** [webspiderteam/Bluetooth-DMM-For-Windows#24] [Announcements]
- Added scrollbar to Sample Output in Mqtt Values window
- Added Show status text when connection status changed
- UI improvements and stablize

<hr /><em>This discussion was created from the release <a href='https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/releases/tag/WPFRelease1.19'>Release 1.19</a>.</em>

**Comment by ludwich66** [comment]
Hello engine,
great that you are so diligent.
The changed MQTT display is great. You can adapt the standard output in a subsequent version.
![image](https://user-images.githubusercontent.com/12202733/184622046-637dd60d-06f9-4c3f-b5c4-62177d9ba3c0.png)

Do you have an idea about the picture?
The MQTT settings window is lost in the window below due to its size, it's really difficult to see which "save" goes where. Can the position be moved so that only the save of the upper window can be seen?
Ist it possible to increase the Window-Border?
![image](https://user-images.githubusercontent.com/12202733/184621096-c07ae647-6b92-4d47-a745-8df84412f5d5.png)

And a Eye candy problem:
Alignement Button "Manage Known Devices" and "MQTT Settings"
Left aligment  as MQTT or under "Auto" / "Enable" 0,5 cm right from checkbox?
![image](https://user-images.githubusercontent.com/12202733/184623205-1adf7155-fd52-4351-b276-3a01227fa58a.png)
![image](https://user-images.githubusercontent.com/12202733/184626519-d98d092f-2494-4a2e-8546-4d866dce4c5b.png)

Idea to uniform occurrence (1. window to Setting Window)
change order like  Menue Window

WindowTitle Settings
more gap to 
Languagesetting:
Language Button aligned left like as the other Buttons
gap to
StartupSettings:
1. Run on Windows Start
1. ask before
2. Auto connect
3. Manage Known
4. gap to
5. Open datachart
6. open Analog display
7. gap to
8. Always on Top
9. save Window Pos
10. Always Minimize to Tray
gap
Log Settings:
1. Log Raw Data (Debugmodus)
2. Set CSV to ...
3. Enable MQTT
4. MQTT Settings

What did you meen? No Problems only structural changes.


Where can i find this?
Added Show status text when connection status changed

Heads Up!

L

**Reply by webspiderteam** [reply]
> You can adapt the standard output in a subsequent version.
> ![image](https://user-images.githubusercontent.com/12202733/184622046-637dd60d-06f9-4c3f-b5c4-62177d9ba3c0.png)

I think its better to keep it simple. Defaults are as Multimeter Display If anyone want to change they can.


> The MQTT settings window is lost in the window below due to its size, it's really difficult to see which "save" goes where. Can the position be moved so that only the save of the upper window can be seen?

Positions are randomly I can set only position fixed. But this cant solve the problem. Maybe I can find a way to make inactive window grayed out.
![image](https://user-images.githubusercontent.com/85828505/184650727-55a64951-e2bd-4eed-99c6-0a079edbebe9.png)

> And a Eye candy problem:
> Alignement Button "Manage Known Devices" and "MQTT Settings"
> Left aligment as MQTT or under "Auto" / "Enable" 0,5 cm right from checkbox?

![image](https://user-images.githubusercontent.com/85828505/184651567-d838d7f8-0ad6-4e6f-83d4-d2a88f1144f5.png)


what do you think now. I dont know but I think there is something strange

**Reply by webspiderteam** [reply]
> Added Show status text when connection status changed

Sorry not added changed. when mouse not over main window Connected string was not shown even if connected or disconnected. Changed it to visible when connection status changed and stays there if connection lost.

**Reply by webspiderteam** [reply]
> What did you meen? No Problems only structural changes.

I dont Have any improvements for this app Also I think i did all that can be done. If there is no problems that shoul be enough for this app.

**Reply by ludwich66** [reply]
-->I think its better to keep it simple. Defaults are as Multimeter Display If anyone want to change they can.
Yes, that's my approach, the essential values as "harmonious" as possible with the CSV output

MQTT Settings Window:
A thicker app border ins not makeable?

Settings Window:
no, the view is much more relaxed with this alignment
Again, I kept an eye on the order in the first window and the different arrangement in the second window.

***
I think we are working on a high level of ergonomics here. I hope you enjoy this. The app is functionally complete.!!
Its just for perfection!
***

**Comment by webspiderteam** [comment]
[BluetoothDMM.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/9338042/BluetoothDMM.zip)
try this. Does it need to change border? I'm thinking of OBS studio users. if they use color transparency border will be there

**Reply by ludwich66** [reply]
Hi Engin,
File is not loadable ?? "Not found" i dont know where is the problem.
Right Klick - save as doesnot work too.
L

**Reply by webspiderteam** [reply]
[BluetoothDMM.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/9338808/BluetoothDMM.zip)
today github has problem

**Reply by ludwich66** [reply]
ok now works
let#s see

**Reply by ludwich66** [reply]
it does not work -  There is a Microsoft Virus Block ??

**Reply by ludwich66** [reply]
![image](https://user-images.githubusercontent.com/12202733/184685968-2d89365f-2c34-4aa9-9459-dcdfc95fee75.png)

**Reply by webspiderteam** [reply]
false alarm. windows startup option writes to registry thats why it warns you

**Reply by ludwich66** [reply]
Is Ok - it is better to differentiat now. (on my screen)

Which color do you select ?
Deep Black and Dark Gray

**Reply by webspiderteam** [reply]
windows dark theme backgroun preset. I dont know name or code

**Reply by webspiderteam** [reply]
also it can be different in every windows

**Reply by webspiderteam** [reply]
color code R="20" G="20" B="20" A="255"

**Reply by ludwich66** [reply]
Thanks for you Virustipp ;-)

Did you have tested MQTT with Password credentials?
I jused it only withouth Securitiy options.
Its  a next Step for tomorrow.

Is everything else okay with you?

Greetings Ludwich

**Reply by webspiderteam** [reply]
No ı havent tested. Yeah it's ok. Just chilling

**Reply by webspiderteam** [reply]
Did you test security login?

**Reply by ludwich66** [reply]
Hi Engin,
no, i didn't test anymore. I interpreted your last mail that you think the test is superfluous.
L

**Reply by webspiderteam** [reply]
I didnt mean it. I just not know much about mqtt and ı dont know how to test it.

**Reply by ludwich66** [reply]
Ok i look tomorrow 
CU

**Reply by webspiderteam** [reply]
Thank you very much :)

**Reply by ludwich66** [reply]
Hello Engine,
I had problems converting my existing unencrypted MQTT platform. I will set up a second broker as soon as possible. I have to get a Raspi free first.
Feel free to publish the latest changes without the test.
I might be away for the weekend...
Greetings Ludwich

**Reply by webspiderteam** [reply]
Have a nice holiday :)

**Comment by ludwich66** [comment]
Not Holiday, But Day

 I found a Minute and Set Up a new MQTT Broker - Now i hope to connect devices.

L
